cloud/aws-ec2/README.md

```diff
@@ -0,0 +1,45 @@
+# Deploying to AWS EC2 using Terraform
```

Comment:
Where shall I add the reference to the guide?

Reply:
https://secrethub.io/img/integrations/aws-ecs/github-banner.png